src/nomad_simulations/schema_packages/basis_set.py

```diff
@@ -190,16 +190,82 @@ class AtomCenteredFunction(ArchiveSection):
     Specifies a single function (term) in an atom-centered basis set.
     """
 
-    pass
+    function_type = Quantity(
+        type=MEnum('S', 'P', 'D', 'F', 'G', 'H', 'I', 'J'),
+        description="""
+        the angular momentum of the shell to be added.
+        """,
+    )
```

Comment:
Looks more like something for `AtomsState`. @JosePizarro3

Reply:
I think @EBB2675 wants to store the angular momentum here simply. I'd say if a string is enough, go ahead. Tho I will keep consistent with notation: small letters `type=MEnum('s', 'p', 'd', 'f')`

Now, are there cases where one needs to go to h, i, j? I must admit g "might" be ok in some excited state calculations, but the others... I am not sure.

Reply:
> Tho I will keep consistent with notation: small letters type=MEnum('s', 'p', 'd', 'f')

Shouldn't we use capital letters for the total momentum?

Reply:
Ah sorry, if it is total ang mom yes, capital letters is good. Tho we need to decide where this kind of information should be living in a more consistent way.

We can leave that discussion out of this pr.